### PR TITLE
feat: health check CLI command

### PR DESCRIPTION
**Pull request by adrysn**
This PR adds support host health check command.

```console
backend.ai manager health-check [--with-all-agents] [agent-id], ...
```

Internal Ticket: OP#664
Related: lablup/backend.ai-manager#351, lablup/backend.ai-agent#241

### CODE REVIEW

**Review by achimnol — CHANGES_REQUESTED** [review]
Please add the news fragment (towncrier changelog).

File: src/ai/backend/client/cli/manager.py [hunk 1]
```diff
@@ -32,6 +32,21 @@ def status():
         sys.exit(1)
 
 
+@manager.command()
+@click.argument('agents', nargs=-1)
+@click.option('-a', '--with-all-agents', is_flag=True,
+              help='Fetch all agent host information as well.')
```
Comment (achimnol) [inline src/ai/backend/client/cli/manager.py:38]: Please update the parameter & cli argument list accordingly to the review comment in the manager PR.

File: changes/145.feature [hunk 2]
```diff
@@ -0,0 +1 @@
+Add host health check command and SDK.
```
Comment (achimnol) [inline changes/145.feature:1]: Let's mention the command, `backend.ai manager health-check`.

**Review by achimnol — CHANGES_REQUESTED** [review]

File: src/ai/backend/client/cli/manager.py (hunk 1 above)
Comment (adrysn) [reply src/ai/backend/client/cli/manager.py]: Changed parameter to conform to the changes in the manager.

File: changes/145.feature (hunk 2 above)
Comment (adrysn) [reply changes/145.feature]: Appended command to the news fragment.

**Comment by achimnol** [comment]
Closing in favor of our new watcher framework.

**Pull request closed without merging**